mteb/tasks/Retrieval/nld/ArguAnaNLRetrieval.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class ArguAnaNL(AbsTaskRetrieval):
+    ignore_identical_ids = True
+
+    metadata = TaskMetadata(
+        name="ArguAna-NL",
+        description="ArguAna involves the task of retrieval of the best counterargument to an argument. ArguAna-NL is "
+        "a Dutch translation.",
+        reference="https://huggingface.co/datasets/clips/beir-nl-arguana",
+        dataset={
+            "path": "clips/beir-nl-arguana",
+            "revision": "4cd085d148fe2cac923bb7758d6ef585926170ba",
+        },
+        type="Retrieval",
+        category="s2p",
+        modalities=["text"],
+        eval_splits=["test"],
+        eval_langs=["nld-Latn"],
+        main_score="ndcg_at_10",
+        date=("2024-10-01", "2024-10-01"),
+        domains=["Written"],
+        task_subtypes=[],
+        license="cc-by-sa-4.0",
+        annotations_creators="LM-generated and reviewed",
+        dialect=[],
+        sample_creation="machine-translated and verified",
```

Comment:
How were these verified?

Reply:
We manually checked a small subset of translations. If it does not fit into the "verified" category. I can remove that.

Reply:
No that is perfectly fine - will you add a comment like so: 
```suggestion
        sample_creation="machine-translated and verified", # manually checked a small subset
```

Reply:
Done.